### ISSUE TITLE
"My Streams" tab broken and can't display anything

### ISSUE THREAD

**Issue by Porkepix** [twireapp/Twire#46]
See joined screenshot:

![Screenshot_20191009-145938](https://user-images.githubusercontent.com/709500/66483678-d70d7e80-eaa5-11e9-8726-ff396fb77762.png)
"My Streams" tab fail to display anything every time I try.

Here are Java exceptions caught into `adb logcat -v threadtime`

```
10-09 14:58:57.363 16791 16791 I Timeline: Timeline: Activity_launch_request id:com.perflyst.twire time:730291226
10-09 14:58:57.365   920  3828 I ActivityManager: START u0 {flg=0x10000 cmp=com.perflyst.twire/.activities.main.MyStreamsActivity (has extras)} from uid 10441 on display 0
10-09 14:58:57.468 16791 16791 E View    : hasTransientState decremented below 0: unmatched pair of setHasTransientState calls
10-09 14:58:57.513   920   978 I ActivityManager: Displayed com.perflyst.twire/.activities.main.MyStreamsActivity: +126ms
10-09 14:58:57.513   920   978 I Timeline: Timeline: Activity_windows_visible id: ActivityRecord{38a9cf5 u0 com.perflyst.twire/.activities.main.MyStreamsActivity t3387} time:730291376
10-09 14:58:57.734 16791 16867 W System.err: java.io.FileNotFoundException: https://api.twitch.tv/kraken/streams/followed?oauth_token=HASH&offset=0&stream_type=live
10-09 14:58:57.735 16791 16867 W System.err: 	at com.android.okhttp.internal.huc.HttpURLConnectionImpl.getInputStream(HttpURLConnectionImpl.java:238)
10-09 14:58:57.735 16791 16867 W System.err: 	at com.android.okhttp.internal.huc.DelegatingHttpsURLConnection.getInputStream(DelegatingHttpsURLConnection.java:210)
10-09 14:58:57.735 16791 16867 W System.err: 	at com.android.okhttp.internal.huc.HttpsURLConnectionImpl.getInputStream(HttpsURLConnectionImpl.java:25)
10-09 14:58:57.735 16791 16867 W System.err: 	at com.perflyst.twire.service.Service.urlToJSONString(Service.java:607)
10-09 14:58:57.735 16791 16867 W System.err: 	at com.perflyst.twire.tasks.GetStreamsCountTask.doInBackground(GetStreamsCountTask.java:29)
10-09 14:58:57.737 16791 16867 W System.err: 	at com.perflyst.twire.tasks.GetStreamsCountTask.doInBackground(GetStreamsCountTask.java:14)
10-09 14:58:57.737 16791 16867 W System.err: 	at android.os.AsyncTask$2.call(AsyncTask.java:295)
10-09 14:58:57.738 16791 16867 W System.err: 	at java.util.concurrent.FutureTask.run(FutureTask.java:237)
10-09 14:58:57.738 16791 16867 W System.err: 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1113)
10-09 14:58:57.738 16791 16867 W System.err: 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:588)
10-09 14:58:57.738 16791 16867 W System.err: 	at java.lang.Thread.run(Thread.java:818)
10-09 14:58:57.743 16791 16867 V URL TO JSON STRING: https://api.twitch.tv/kraken/streams/followed?oauth_token=HASH&offset=0&stream_type=live did not successfully get read
10-09 14:58:57.743 16791 16867 V URL TO JSON STRING: Result of reading -
10-09 14:58:57.745 16791 16867 W System.err: org.json.JSONException: End of input at character 0 of
10-09 14:58:57.745 16791 16867 W System.err: 	at org.json.JSONTokener.syntaxError(JSONTokener.java:449)
10-09 14:58:57.745 16791 16867 W System.err: 	at org.json.JSONTokener.nextValue(JSONTokener.java:97)
10-09 14:58:57.745 16791 16867 W System.err: 	at org.json.JSONObject.<init>(JSONObject.java:156)
10-09 14:58:57.746 16791 16867 W System.err: 	at org.json.JSONObject.<init>(JSONObject.java:173)
10-09 14:58:57.746 16791 16867 W System.err: 	at com.perflyst.twire.tasks.GetStreamsCountTask.doInBackground(GetStreamsCountTask.java:30)
10-09 14:58:57.746 16791 16867 W System.err: 	at com.perflyst.twire.tasks.GetStreamsCountTask.doInBackground(GetStreamsCountTask.java:14)
10-09 14:58:57.746 16791 16867 W System.err: 	at android.os.AsyncTask$2.call(AsyncTask.java:295)
10-09 14:58:57.746 16791 16867 W System.err: 	at java.util.concurrent.FutureTask.run(FutureTask.java:237)
10-09 14:58:57.746 16791 16867 W System.err: 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1113)
10-09 14:58:57.746 16791 16867 W System.err: 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:588)
10-09 14:58:57.746 16791 16867 W System.err: 	at java.lang.Thread.run(Thread.java:818)
10-09 14:58:57.902 16791 16813 W System.err: java.io.FileNotFoundException: https://api.twitch.tv/kraken/streams/followed?oauth_token=HASH&limit=10&offset=0&stream_type=live
10-09 14:58:57.902 16791 16813 W System.err: 	at com.android.okhttp.internal.huc.HttpURLConnectionImpl.getInputStream(HttpURLConnectionImpl.java:238)
10-09 14:58:57.903 16791 16813 W System.err: 	at com.android.okhttp.internal.huc.DelegatingHttpsURLConnection.getInputStream(DelegatingHttpsURLConnection.java:210)
10-09 14:58:57.903 16791 16813 W System.err: 	at com.android.okhttp.internal.huc.HttpsURLConnectionImpl.getInputStream(HttpsURLConnectionImpl.java:25)
10-09 14:58:57.903 16791 16813 W System.err: 	at com.perflyst.twire.service.Service.urlToJSONString(Service.java:607)
10-09 14:58:57.903 16791 16813 W System.err: 	at com.perflyst.twire.activities.main.MyStreamsActivity.getVisualElements(MyStreamsActivity.java:60)
10-09 14:58:57.903 16791 16813 W System.err: 	at com.perflyst.twire.tasks.GetVisualElementsTask.doInBackground(GetVisualElementsTask.java:29)
10-09 14:58:57.903 16791 16813 W System.err: 	at com.perflyst.twire.tasks.GetVisualElementsTask.doInBackground(GetVisualElementsTask.java:11)
10-09 14:58:57.903 16791 16813 W System.err: 	at android.os.AsyncTask$2.call(AsyncTask.java:295)
10-09 14:58:57.903 16791 16813 W System.err: 	at java.util.concurrent.FutureTask.run(FutureTask.java:237)
10-09 14:58:57.903 16791 16813 W System.err: 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1113)
10-09 14:58:57.903 16791 16813 W System.err: 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:588)
10-09 14:58:57.903 16791 16813 W System.err: 	at java.lang.Thread.run(Thread.java:818)
10-09 14:58:57.904 16791 16813 V URL TO JSON STRING: https://api.twitch.tv/kraken/streams/followed?oauth_token=HASH&limit=10&offset=0&stream_type=live did not successfully get read
10-09 14:58:57.904 16791 16813 V URL TO JSON STRING: Result of reading -
10-09 14:58:57.905 16791 16813 W System.err: org.json.JSONException: End of input at character 0 of
10-09 14:58:57.905 16791 16813 W System.err: 	at org.json.JSONTokener.syntaxError(JSONTokener.java:449)
10-09 14:58:57.905 16791 16813 W System.err: 	at org.json.JSONTokener.nextValue(JSONTokener.java:97)
10-09 14:58:57.905 16791 16813 W System.err: 	at org.json.JSONObject.<init>(JSONObject.java:156)
10-09 14:58:57.905 16791 16813 W System.err: 	at org.json.JSONObject.<init>(JSONObject.java:173)
10-09 14:58:57.905 16791 16813 W System.err: 	at com.perflyst.twire.activities.main.MyStreamsActivity.getVisualElements(MyStreamsActivity.java:61)
10-09 14:58:57.905 16791 16813 W System.err: 	at com.perflyst.twire.tasks.GetVisualElementsTask.doInBackground(GetVisualElementsTask.java:29)
10-09 14:58:57.905 16791 16813 W System.err: 	at com.perflyst.twire.tasks.GetVisualElementsTask.doInBackground(GetVisualElementsTask.java:11)
10-09 14:58:57.905 16791 16813 W System.err: 	at android.os.AsyncTask$2.call(AsyncTask.java:295)
10-09 14:58:57.905 16791 16813 W System.err: 	at java.util.concurrent.FutureTask.run(FutureTask.java:237)
10-09 14:58:57.905 16791 16813 W System.err: 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1113)
10-09 14:58:57.905 16791 16813 W System.err: 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:588)
10-09 14:58:57.905 16791 16813 W System.err: 	at java.lang.Thread.run(Thread.java:818)
10-09 14:58:57.910 16791 16791 I GetVisualElementsTask: ADDING 0 VISUAL ELEMENTS
10-09 14:58:57.910 16791 16791 I         : Adding Top Streams: 0
10-09 14:58:58.152 16791 16811 W System.err: java.io.FileNotFoundException: https://api.twitch.tv/kraken/streams/followed?oauth_token=HASH&limit=10&offset=0&stream_type=live
10-09 14:58:58.154 16791 16811 W System.err: 	at com.android.okhttp.internal.huc.HttpURLConnectionImpl.getInputStream(HttpURLConnectionImpl.java:238)
10-09 14:58:58.154 16791 16811 W System.err: 	at com.android.okhttp.internal.huc.DelegatingHttpsURLConnection.getInputStream(DelegatingHttpsURLConnection.java:210)
10-09 14:58:58.154 16791 16811 W System.err: 	at com.android.okhttp.internal.huc.HttpsURLConnectionImpl.getInputStream(HttpsURLConnectionImpl.java:25)
10-09 14:58:58.154 16791 16811 W System.err: 	at com.perflyst.twire.service.Service.urlToJSONString(Service.java:607)
10-09 14:58:58.154 16791 16811 W System.err: 	at com.perflyst.twire.activities.main.MyStreamsActivity.getVisualElements(MyStreamsActivity.java:60)
10-09 14:58:58.155 16791 16811 W System.err: 	at com.perflyst.twire.tasks.GetVisualElementsTask.doInBackground(GetVisualElementsTask.java:29)
10-09 14:58:58.155 16791 16811 W System.err: 	at com.perflyst.twire.tasks.GetVisualElementsTask.doInBackground(GetVisualElementsTask.java:11)
10-09 14:58:58.155 16791 16811 W System.err: 	at android.os.AsyncTask$2.call(AsyncTask.java:295)
10-09 14:58:58.155 16791 16811 W System.err: 	at java.util.concurrent.FutureTask.run(FutureTask.java:237)
10-09 14:58:58.155 16791 16811 W System.err: 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1113)
10-09 14:58:58.155 16791 16811 W System.err: 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:588)
10-09 14:58:58.155 16791 16811 W System.err: 	at java.lang.Thread.run(Thread.java:818)
10-09 14:58:58.155 16791 16811 V URL TO JSON STRING: https://api.twitch.tv/kraken/streams/followed?oauth_token=HASH&limit=10&offset=0&stream_type=live did not successfully get read
10-09 14:58:58.155 16791 16811 V URL TO JSON STRING: Result of reading -
10-09 14:58:58.155 16791 16811 W System.err: org.json.JSONException: End of input at character 0 of
10-09 14:58:58.156 16791 16811 W System.err: 	at org.json.JSONTokener.syntaxError(JSONTokener.java:449)
10-09 14:58:58.156 16791 16811 W System.err: 	at org.json.JSONTokener.nextValue(JSONTokener.java:97)
10-09 14:58:58.156 16791 16811 W System.err: 	at org.json.JSONObject.<init>(JSONObject.java:156)
10-09 14:58:58.156 16791 16811 W System.err: 	at org.json.JSONObject.<init>(JSONObject.java:173)
10-09 14:58:58.156 16791 16811 W System.err: 	at com.perflyst.twire.activities.main.MyStreamsActivity.getVisualElements(MyStreamsActivity.java:61)
10-09 14:58:58.156 16791 16811 W System.err: 	at com.perflyst.twire.tasks.GetVisualElementsTask.doInBackground(GetVisualElementsTask.java:29)
10-09 14:58:58.156 16791 16811 W System.err: 	at com.perflyst.twire.tasks.GetVisualElementsTask.doInBackground(GetVisualElementsTask.java:11)
10-09 14:58:58.156 16791 16811 W System.err: 	at android.os.AsyncTask$2.call(AsyncTask.java:295)
10-09 14:58:58.156 16791 16811 W System.err: 	at java.util.concurrent.FutureTask.run(FutureTask.java:237)
10-09 14:58:58.156 16791 16811 W System.err: 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1113)
10-09 14:58:58.156 16791 16811 W System.err: 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:588)
10-09 14:58:58.156 16791 16811 W System.err: 	at java.lang.Thread.run(Thread.java:818)
10-09 14:58:58.163 16791 16791 I GetVisualElementsTask: ADDING 0 VISUAL ELEMENTS
10-09 14:58:58.163 16791 16791 I         : Adding Top Streams: 0
```

**Comment by widmark-takeit** [comment]
I had the same problem, logged out and in again and it started working.

**Comment by samfundev** [comment]
I believe this is fixed by ea864e98be29471cdea610119176b7f9039f6e78, which will check to see if you need to login again.
